modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/RatedRequest.java

```diff
@@ -60,16 +60,28 @@
     private SearchSourceBuilder testRequest;
     /** Map of parameters to use for filling a query template, can be used instead of providing testRequest. */
     private Map<String, Object> params = new HashMap<>();
+    @Nullable
+    private String templateId;
 
-    public RatedRequest(String id, List<RatedDocument> ratedDocs, SearchSourceBuilder testRequest, Map<String, Object> params) {
+    // TODO include template id in sanity checks
```

Comment:
Looking at the null/ empty string checks that follow I believe this is done, removing the comment.